main.go

```diff
@@ -374,9 +380,15 @@ func initWebhookServer() {
 			tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
 		}
 	}
+	clientCfgFunc := func(cfg *tls.Config) {
```

Comment:
Can we move it one block down so it is declared where it is used?  Tripped me a little trying to figure out why it was here.